src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -422,6 +422,12 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'tags': ['experimental'],
 				'default': 'both',
 				'description': localize('layoutControlType', "Controls whether the layout control in the custom title bar is displayed as a single menu button or with multiple UI toggles."),
+			},
+			'workbench.experimental.dragAndDropIntoEditor.enabled': {
```

Comment:
Later the settings name should be prefixed `workbench.editor` to group it with other related editor settings.

Reply:
That makes sense. I'll update the experimental setting name to be closer to the final name